libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx

```diff
@@ -179,6 +179,10 @@ export const reducer = (
         activeFab: action.activeFab
       }
     case 'SetActiveSlideAction':
+      if (state.showAnalytics === true) {
+        return { ...state, activeSlide: 0 }
+      }
+
```

Comment:
Remove the if statement and replace the return with this
```suggestion
        return { ...state, 
            activeSlide: state.showAnalytics === true ? 0 : action.activeSlide
```